Dockerfile-boot

```diff
@@ -11,7 +11,7 @@ LABEL maintainer="jenkins-x"
 RUN addgroup -S app \
     && adduser -S -g app app \
     && apk --no-cache add \
-    ca-certificates curl git git-lfs make netcat-openbsd bash
+    ca-certificates curl git git-lfs make netcat-openbsd bash yq
```

Comment:
minor NIT: May be we can have each package in a separate line to improve readability.